woocommerce/woocommerce
# Remove `wc-admin-set-up-additional-payment-types` inbox note

### Changes proposed in this Pull Request:



Closes https://github.com/woocommerce/woocommerce/issues/32719. Removes the said note.

### How to test the changes in this Pull Request:

1. If your store doesn't have "Set up additional providers" note, do the following:
   1. Check out `trunk`
   3. Go to setup wizard, select WooCommerce Payments in business step
   5. Go to WooCommerce > Home
   6. Observe the note exists
2. Simulate WC installation by doing:
    1.  Go to `wp-admin/options.php`
    2. Search `woocommerce_version`
    3. Change it to `6.5.0`
3. Go to WooCommerce > Home
4. Observe the note is deleted
5. Run `wc_admin_daily` cronjob
6. Make sure no errors appear in `wp-content/debug.log`

### Other information:




### FOR PR REVIEWER ONLY:
